scripts/run_cuda_test.sh

```diff
@@ -17,14 +17,17 @@
 set -o errexit
 set -o nounset
 
-CUDA_TEST_CONTAINER="gcr.io/google_containers/cuda-vector-add:v0.1"
 GPU_INSTALLER_ENV_KEY="cos-gpu-installer-env"
 GPU_INSTALLER_ENV_PATH="/etc/gpu-installer-env"
+# The following environment variables may be changed by cos-gpu-installer-env.
+NVIDIA_INSTALL_DIR_HOST="/var/lib/nvidia"
```

Comment:
Do you want to comment these out (or even remove) from the `scripts/gpu-installer-env` file?